src/Workleap.Extensions.OpenAPI/TypedResult/ExtractSchemaTypeResultFilter.cs

```diff
@@ -8,23 +8,31 @@ internal sealed class ExtractSchemaTypeResultFilter : IOperationFilter
 {
     // Based on this documentation: https://learn.microsoft.com/en-us/aspnet/core/web-api/advanced/formatting?view=aspnetcore-8.0
     private const string DefaultContentType = "application/json";
+    private const string ProducesResponseTypeAttribute = "Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute";
+    private const string SwaggerResponseTypeAttribute = "Swashbuckle.AspNetCore.Annotations.SwaggerResponseAttribute";
 
     public void Apply(OpenApiOperation operation, OperationFilterContext context)
     {
+        // If the endpoint has annotations defined, we don't want to remove them.
+        var responseCodes = ExtractResponseCodesFromAttributes(context);
```

Comment:
We could rename to explicitDefinedResponseCodes for more explicit explication of the purpose.